### PR TITLE
 [vi] translation

### PR DESCRIPTION
**Pull request by ntvan711**

### CODE REVIEW

File: locales/vi/vi.json [hunk 1]
```diff
@@ -127,7 +127,7 @@
     "Choose Type": "Chọn Loại",
     "Christmas Island": "Đảo Giáng Sinh",
     "City": "Thành Phố",
-    "click here to request another": "click here to request another",
+    "click here to request another": "Click vào đây để tạo yêu cầu khác",
```
Comment (andrey-helldar) [inline locales/vi/vi.json:130]: Is it correct that in your version the sentence starts with a capital letter?

File: locales/vi/vi.json [hunk 2]
```diff
@@ -272,7 +272,7 @@
     "If you already have an account, you may accept this invitation by clicking the button below:": "Nếu bạn đã có tài khoản, bạn có thể chấp nhận lời mời này bằng cách bấm vào nút bên dưới:",
     "If you did not create an account, no further action is required.": "Nếu bạn không đăng ký tài khoản này, bạn không cần thực hiện thêm hành động nào.",
     "If you did not expect to receive an invitation to this team, you may discard this email.": "Nếu bạn không muốn chấp nhận lời mời, bạn có thể bỏ qua email này.",
-    "If you did not receive the email": "If you did not receive the email",
+    "If you did not receive the email": "Nếu bạn không nhận được email.",
```
Comment (andrey-helldar) [inline locales/vi/vi.json:275]:
```suggestion
    "If you did not receive the email": "Nếu bạn không nhận được email",
```

The full stop at the end of the sentence is redundant.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: locales/vi/validation-inline.php [hunk 3]
```diff
@@ -35,16 +35,16 @@
     'date'                 => 'Trường này không phải là định dạng của ngày-tháng.',
     'date_equals'          => 'Trường này phải là một ngày bằng với :date.',
     'date_format'          => 'Trường này không giống với định dạng :format.',
-    'declined'             => 'This value must be declined.',
-    'declined_if'          => 'This value must be declined when :other is :value.',
+    'declined'             => 'Giá trị này phải bị từ chối..',
```
Comment (andrey-helldar) [inline locales/vi/validation-inline.php:38]:
```suggestion
    'declined'             => 'Giá trị này phải bị từ chối.',
```
Extra full stop at the end of a sentence.

File: locales/vi/validation-inline.php [hunk 4]
```diff
@@ -80,7 +80,7 @@
         'numeric' => 'Giá trị trường này phải nhỏ hơn hoặc bằng :value.',
         'string'  => 'Độ dài trường này phải nhỏ hơn hoặc bằng :value kí tự.',
     ],
-    'mac_address'          => 'The value must be a valid MAC address.',
+    'mac_address'          => 'Giá trị phải là địa chỉ MAC hợp lệ',
```
Comment (andrey-helldar) [inline locales/vi/validation-inline.php:83]: Missing full stop at the end of a sentence.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

File: locales/vi/vi.json [hunk 5]
```diff
@@ -127,7 +127,7 @@
     "Choose Type": "Chọn Loại",
     "Christmas Island": "Đảo Giáng Sinh",
     "City": "Thành Phố",
-    "click here to request another": "click here to request another",
+    "click here to request another": "click vào đây để tạo yêu cầu khác",
```
[the remark this answers is not in the document]
Comment (dinhquochan) [reply locales/vi/vi.json]: Yes @andrey-helldar, Its mean "Nhấn"/ "Bấm" but "Click" is common.

File: locales/vi/vi.json [hunk 6]
```diff
@@ -190,7 +190,7 @@
     "Done.": "Xong.",
     "Download": "Tải Về",
     "Download Receipt": "Tải Về Hóa Đơn",
-    "E-Mail Address": "E-Mail Address",
+    "E-Mail Address": "Địa chỉ Emails",
```
Comment (dinhquochan) [inline locales/vi/vi.json:193]: It's should be "Email" and upcasse "C" letter.

File: locales/vi/vi.json [hunk 7]
```diff
@@ -673,7 +673,7 @@
     "Venezuela": "Venezuela",
     "Venezuela, Bolivarian Republic of": "Venezuela, Công Hòa Bolivar",
     "Verify Email Address": "Xác Minh Địa Chỉ Email",
-    "Verify Your Email Address": "Verify Your Email Address",
+    "Verify Your Email Address": "Xác minh địa chỉ email của bạn",
```
Comment (dinhquochan) [inline locales/vi/vi.json:676]: Upcase all first letters.

**Review by dinhquochan — CHANGES_REQUESTED** [review]

**Review by dinhquochan — APPROVED** [review]
Nice, Thanks @ntvan711

**Comment by andrey-helldar** [comment]
Obrigado

**Pull request merged**